digitalbitbox/bitbox-wallet-app
# frontend: fix broken svgs

Some of the svg didn't render anymore, this was a regression
introduced in https://github.com/digitalbitbox/bitbox-wallet-app/pull/1531/commits/6a7d5db395bf1dc25bec5e64a6454463e2ea4dee

How to test:
- build Qt app
- unlock with BitBox02 Multi
- goto Settings > Manage Accounts
- Enable at least 1 Ethereum account
- toggle show tokens below to see the tokens
